spring-cloud-circuitbreaker-spring-retry/src/main/java/org/springframework/cloud/circuitbreaker/springretry/SpringRetryCircuitBreaker.java

```diff
@@ -34,12 +33,12 @@
 
 	private final SpringRetryConfig config;
 
-	private final Optional<Customizer<RetryTemplate>> retryTemplateCustomizer;
+	private final Customizer<RetryTemplate> retryTemplateCustomizer;
 
 	private final RetryTemplate retryTemplate;
 
 	public SpringRetryCircuitBreaker(String id, SpringRetryConfig config,
```

Comment:
The fix in general is about not to use Optional parameters, it was not designed for this and in the java world is seen as a miss-use. Not a big deal of course, but people look at our code base and think "this is fine". [here](https://stackoverflow.com/questions/23454952/uses-for-optional) is Stuart Marks talking more about this (and much better then I ever could).

This is still very opinion based, I agree. Let me know what you think about it.

---

Since this constructor is `public`, this is a breaking change, so I assume it's fine to go into `main`?